src/lib.rs

```diff
@@ -173,13 +177,16 @@ enum OfferState {
 pub struct OfferHandler {
     _active_offers: Mutex<HashMap<String, OfferState>>,
     pending_messages: Mutex<Vec<PendingOnionMessage<OffersMessage>>>,
+    // This channel will close once the onion messenger has successfully started up.
+    _started: RefCell<Receiver<u32>>,
```

Comment:
Do we need to store this channel in the OfferHandler? If we just need it once off for startup, can't it juse be passed in once-off to wherever we do that wait?

Reply:
Sure that makes sense

Reply:
Good call